pygfx/renderers/wgpu/shaders/pointsshader.py

```diff
@@ -76,8 +76,10 @@ def __init__(self, wobject):
             self["edge_color_mode"] = "uniform"
             self["edge_color_buffer_channels"] = 0
 
-        self["edge_color_mode"] = edge_color_mode
```

Comment:
did you intentionally remove this line?

Reply:
notice that it is defined in both "if/else" paths

Reply:
gotcha!